### DISCUSSION TITLE
Release pytest-asyncio 0.23.3

### DISCUSSION THREAD

**Discussion by psalaberria002** [pytest-dev/pytest-asyncio#723] [General]
The pre-release for [pytest-asyncio 0.23.3a0](https://github.com/pytest-dev/pytest-asyncio/releases/tag/v0.23.3a0) has been out for a week. Mind releasing it?

Thank you.

**Comment by seifertm** [comment]
A number of other issues related to test collection came up so one of the fixes needed to be replaced with a more sustainable solution.

The patch release will be out in a couple of days. May I ask which issue has been resolved for you by the alpha release?
